Modules/Messages/test/Messages.Domain.Tests/Messages/MessagesTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Messages.Domain.DomainEvents.Outgoing;
+using Backbone.Modules.Messages.Domain.Entities;
+using Backbone.Modules.Messages.Domain.Ids;
+using Backbone.UnitTestTools.BaseClasses;
+using Backbone.UnitTestTools.Data;
+using Backbone.UnitTestTools.FluentAssertions.Extensions;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Messages.Domain.Tests.Messages;
+
+public class MessagesTests : AbstractTestsBase
+{
+    [Fact]
+    public void Raises_MessageCreatedDomainEvent_when_created()
+    {
+        // Arrange
+        var sender = TestDataGenerator.CreateRandomIdentityAddress();
+        var recipient = TestDataGenerator.CreateRandomIdentityAddress();
+
+        // Act
+        var message = CreateMessage(sender, [recipient]);
```

Comment:
You shouldn't use factory methods to run the code under test.

```suggestion        
        var recipient = new RecipientInformation(TestDataGenerator.CreateRandomIdentityAddress(), RelationshipId.New(), [])

        // Act
        var message = new Message(
            createdBy,
            TestDataGenerator.CreateRandomDeviceId(),
            [],
            [],
            [recipient]
        );
```